l10n_es_facturae/migrations/15.0.1.10.0/pre-migration.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2024 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(env):
+    childs = env["res.partner"].search([("parent_id", "!=", False)])
```

Comment:
Only change those that doesn't match with the parent one. Do it by SQL.

Reply:
Done!